openreview/profile/management.py

```diff
@@ -219,4 +220,105 @@ def set_archive_invitations(self):
                     }
                 }
             }
-        ))                       
+        ))
+
+    def set_merge_profiles_invitations(self):
+
+        content = {
+            'name': {
+                'order': 1,
+                'description': 'Name that want to be removed.',
+                'value-regex': '.*',
+                'required': True                
+            },
```

Comment:
Why are we including this field in this invitation?

Reply:
sometimes guest users are requesting merge of profiles, and we would need to have a name.

Reply:
maybe this should be email instead of name
so that the user can be contacted if we need clarification

Reply:
we discussed about combining signature and email column
so this field need to be renamed to email and required set to false